modules/lambdas/main.tf

```diff
@@ -409,6 +457,55 @@ resource "aws_lambda_function" "orca_catalog_reporting_dummy" {
 }
 
 
+# post_to_catalog - Posts provider/collection/granule/file info from SQS queue to database.
+# ===========================================================================================
+resource "aws_lambda_function" "post_to_catalog" {
+  ## REQUIRED
+  function_name = "${var.prefix}_post_to_catalog"
```

Comment:
check line 274. Looks like both the lambdas have the same name. Is this a duplicate?